packages/ag-charts-community/src/chart/series/cartesian/lineSeries.ts

```diff
@@ -59,6 +59,26 @@ class LineSeriesLabel extends Label {
 type LineContext = SeriesNodeDataContext<LineNodeDatum>;
 type LineAnimationData = CartesianAnimationData<LineContext>;
 
+// TODO(olegat) move somewhere else?
```

Comment:
Please advise whether these kind of `// TODO` comments are acceptable. If not, then I'll make a note of this somewhere else.

Reply:
We don't typically use these - there is a JIRA plugin which adds checklist support, so for smaller things like this I would usually capture them there, assuming you are going to resolve this in the scope of your current ticket?

Reply:
Understood. I'll remove this comment.